certification/formatters/formatters_test.go

```diff
@@ -46,7 +47,7 @@ var _ = Describe("Formatters", func() {
 	Describe("When creating a new generic formatter", func() {
 		Context("with improper arguments", func() {
 			expectedResult := []byte(fmt.Errorf("failed to create a new generic formatter: formatter name is required").Error())
-			var fn FormatterFunc = func(context.Context, runtime.Results) ([]byte, error) {
+			fn := func(context.Context, runtime.Results) ([]byte, error) {
```

Comment:
I tried it. Since we are using it in a call just a few lines down, it will become a compile-time error if you don't have the correct signature. I was very cognizant of it when I first did it.

Reply:
> it will become a compile-time error if you don't have the correct signature

Right. Something about declaring the function with an explicit type assignment feels better to me.

If we can't do anything about it, then I suppose it's fine. Just not my favorite change here. Non-blocking, I suppose. I'll let you resolve to move forward.

Reply:
I'll add it back for clarity, and nolint it.